gammapy/makers/utils.py

```diff
@@ -110,6 +130,10 @@ def make_map_background_irf(pointing, ontime, bkg, geom, oversampling=None):
         Reference geometry
     oversampling: int
         Oversampling factor in energy, used for the background model evaluation.
+    average_over_region: Bool
+        If geom is a RegionGeom, whether to just
+        consider the values at the region center
+        or the average over the whole region
```

Comment:
Here it is not an average but a sum. Maybe the argument could reflect that.

Reply:
hm good point. Do you mean changing the name to `sum_over_region` for this case or just changing the description to "sum over the whole region" and keep average in the name? I could see how changing the name could be confusing since the flag that gets passed would still be called `average_over_region`....

Reply:
This part I'm not sure about either...I think it's definitely good to adapt the description. In general we will have of course only one argument on the `Maker` and I agree for consistency we should choose the same name everywhere. However maybe there is a name that matches both cases approximately? Possibly `sum_over_region` is slightly better? In the keyword description of the `Maker` we mention the different cases anyway...

Reply:
Another possibility is to name the other case: in a sense summing / averaging over the region I the correct thing to do and just taking the value at the center of the region is an approximation. So one could call the option e.g. `use_region_center=True` as well...and check for `if not use_region_center:`

Reply:
I think I like `use_region_center` better actually - it is more accurate and I think that should be the case which choice is made explicit, not the most correct one

Reply:
Yes indeed that is a better choice.

Reply:
should the default behavior then be `use_region_center=True` (like it is right now) or should we change it?

Reply:
I think we should definitely keep the `use_region_center=True` in this PR. However just to get a feel on the effect of this option, I'd be interested to see how our analysis results / validation results change when the averaging / integration is applied somehow. I don't expect large changes, but who knows...

Reply:
From what I've seen of how much the test values changes, it is quite a small effect. You can see it if you compare the values for the background from the test I implemented with those of the test above, which is the same dataset but without averaging...

Reply:
Ah, indeed in this case the effect is smaller than 1%...